composer.json

```diff
@@ -14,10 +14,20 @@
     "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
     "ext-zip": "*",
     "pimcore/pimcore": "^11.0",
-    "teamneusta/converter-bundle": "^1.8.0"
+    "symfony/config": "^5.4 || ^6.0 || ^7.0",
+    "symfony/console": "^5.4 || ^6.0 || ^7.0",
+    "symfony/dependency-injection": "^5.4 || ^6.0 || ^7.0",
+    "symfony/filesystem": "^5.4 || ^6.0 || ^7.0",
+    "symfony/finder": "^5.4 || ^6.0 || ^7.0",
+    "symfony/http-foundation": "^5.4 || ^6.0 || ^7.0",
+    "symfony/property-access": "^5.4 || ^6.0 || ^7.0",
+    "symfony/routing": "^5.4 || ^6.0 || ^7.0",
+    "symfony/serializer": "^5.4 || ^6.0 || ^7.0",
+    "symfony/yaml": "^5.4 || ^6.0 || ^7.0",
```

Comment:
Since Symfony `6.0`–`6.3` have not been maintained for at least 1.5 years, I suggest changing it to the LTS version `^6.4`.

Reply:
Done